src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/PageHeader.kt

```diff
@@ -18,6 +23,14 @@ fun BODY.pageHeader(viewModel: HeaderBarViewModel) {
         }
         div(classes = "navbar-menu has-site-branding") {
             div(classes = "navbar-end") {
+                div(classes = "navbar-item") {
+                    input(classes = "input is-small is-rounded has-site-branding") {
+                        id = "search"
+                        type = InputType.search
+                        placeholder = "Search (3 characters required)..."
```

Comment:
Currently, the search box is not wide enough for all this text to be visible:

![image](https://user-images.githubusercontent.com/3872163/231366510-d436ba65-774b-4a07-a439-2543feb981c2.png)